docs/DeveloperGuide.md

```diff
@@ -395,37 +405,17 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **Extensions**
 
-* 1a. There are no events in that year.
+- 2a. The list is empty.
 
   Use case ends.
-  
-* 1b. The given year is invalid.
 
-    * 1a1. Focuris shows an error message.
-
-      Use case resumes at step 1.
+- 3a. The given index is invalid.
 
 **Use case: Reset list of events to current date**
 
-**MSS**
```

Comment:
Shouldn't there be a MSS for this where you state that you request for the list of events to be set to the current date?